package.json

```diff
@@ -16,5 +16,15 @@
   "keywords": [
     "vue",
     "analytics"
-  ]
+  ],
+  "eslintConfig": {
+    "ignorePatterns": [
+      "**/src/*.js"
+    ]
+  },
+  "dependencies": {
+    "axios": "^0.21.1",
+    "bowser": "^2.11.0",
+    "firebase": "^8.6.7"
```

Comment:
H (Hint): Maybe can somehow make those dependencies optional? Maybe [peerDependencies](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependencies) can be used for that. In case this package will be used without `infermedicaAnalytics` the dependencies will not be installed.

Later in the code we can use dynamic imports: https://2ality.com/2017/01/import-operator.html -- which in general IMO should be used (even if we leave the current form of dependencies) to reduce the final build size for cases where `infermedicaAnalytics` is not used.

Reply:
peerDependencies should ok, but users must remember about install this dependencies in there projects :D